chips/stm32h7.toml

```diff
@@ -96,6 +96,14 @@ size = 0x20000
 address = 0x48021800
 size = 4096
 
+[flash_controller]
+address = 0x52002000
+size = 0x20000
```

Comment:
Is this deliberately claiming a larger block than just the flash interface registers?

<img width="218" alt="Screen Shot 2022-03-30 at 9 38 56 AM" src="https://user-images.githubusercontent.com/745333/160847984-5a751912-f174-483e-8eae-d8aa5ec6958c.png">

Reply:
I think this is a typo or something else on my end